static/bidder-info/inmobi.yaml

```diff
@@ -12,8 +12,9 @@ capabilities:
     mediaTypes:
       - banner
       - video
+      - native
```

Comment:
Please add json test case for this change.

Reply:
Hi @gargcreation1992 , I have added the testcase. Kindly help. Many Thanks